File: ruby/A+B.rb
if __FILE__ == $0
  num = 1
  sum = 0
  while num >= 0 do
    num = gets.chomp
    num = /[\.].*\d/.match(num).to_s
    num = /\d/.match(num).to_s.to_i
    sum = sum + num
  end
  puts sum
end

File: c++/A+B.cpp
#include<iostream>
using namespace std;

int main(){
  float num = 0;
  int aux = 0;
  int sum = 0;
  int i = 0;
  while(i < 4){
    cin>>num;
    aux = static_cast<int>(num);
    sum = sum + (((num-aux)+0.01)*10);
    i++;
  }
  cout<<sum<<endl;
  return 0;
}
